panda/src/pnmimage/pnmimage_base.cxx

```diff
@@ -33,7 +33,7 @@ pm_message(const char *format, ...) {
 
   static const size_t buffer_size = 1024;
   char buffer[buffer_size];
-#if defined(WIN32_VC) || defined(WIN64_VC)
+#ifdef _WIN32
```

Comment:
Looks like an outdated concern as [vsprintf is very much a thing in recent VS (at least as far back as 2013)](https://docs.microsoft.com/en-us/previous-versions/visualstudio/visual-studio-2013/1kt27hek%28v%3dvs.120%29). Should we just remove this clause?

Barring that, should be `_MSC_VER` because this is a C runtime concern.